File: backend/src/routes/chatgpt.py
from flask import request
from ..service import chatgpt


def get_awnser_from_chatgpt():
    content = request.json
    keyword_type = content['keyword_type']
    keyword = content['keyword']
    return chatgpt.get_awnser_from_chatgpt(keyword_type, keyword)

File: frontend/src/services/chatgpt.ts
import { axiosInstance } from './axios';

// 从chatgpt获取信息
export const getAwnserFromChatgpt = (keywordType: string, keyword: string) => {
    return axiosInstance.post(`/api/chatgpt/get`, {
        keyword_type: keywordType,
        keyword: keyword,
    });
};
